include/rocksdb/advanced_options.h

```diff
@@ -71,9 +78,10 @@ struct CompactionOptionsFIFO {
   bool allow_compaction = false;
 
   CompactionOptionsFIFO() : max_table_files_size(1 * 1024 * 1024 * 1024) {}
-  CompactionOptionsFIFO(uint64_t _max_table_files_size,
-                        uint64_t _allow_compaction)
+  CompactionOptionsFIFO(uint64_t _max_table_files_size, uint64_t _ttl,
+                        bool _allow_compaction)
```

Comment:
I did start with putting the ttl at the end of the fields, but it breaks one of our tests where we are checking for the struct being compactly aligned I think. 
As part of that change, I moved the ttl param of the constructor to be in the order of the fields declaration. I do remember spending a little while thinking about putting at the end, but somehow I moved the order. Anyway, you have a good point, and I will change it.